pexpect/popen_spawn.py

```diff
@@ -115,21 +115,20 @@ def writelines(self, sequence):
             self.send(s)
 
     def _send(self, s):
-        return self.proc.stdin.write(s)
+        self.proc.stdin.write(s)
 
     def send(self, s):
         s = self._coerce_send_string(s)
         self._log(s, 'send')
 
-        return self._send(s)
+        self._send(s)
```

Comment:
I think the return should be left on these, because the send() method on the existing spawn class is documented as returning the number of bytes written.